### DISCUSSION TITLE
Dexcom One+, Xiaomi 10, latest nightly (07.08.24) - Initializing errors

### DISCUSSION THREAD

**Discussion by kresimirjedud** [NightscoutFoundation/xDrip#3624] [Bug reports]
Hi, 
Everything worked OK until 2 hours ago. The sensor shows readings on the Dexcom reader, so I guess this is something connected to xdrip. The settings are all set as described here: https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html
I tried also this but no help: 
https://navid200.github.io/xDrip/docs/Intermittent.html
Thank you for your help!

![IMG-20240813-WA0016](https://github.com/user-attachments/assets/2fe438ad-1e7a-4b37-9126-b5ac72de2855)

**Comment by Navid200** [comment]
xDrip or your phone.
Considering xDrip works fine with many other phones, it has to be your phone.  

This goes over everything we have as far as troubleshooting: https://navid200.github.io/xDrip/docs/Intermittent.html  

If you use Dexcom app, does it work fine?

**Comment by kresimirjedud** [comment]
I don't use Dexcom app since it is written in the instructions that it
cannot work with both. The sensor worked fine for about 1,5 days and this
happened all of a sudden.
Regards,
Krešimir

uto, 13. kol 2024. 22:53 Navid ***@***.***> je napisao:

**Reply by Navid200** [reply]
You can disable collection in xDrip: https://navid200.github.io/xDrip/docs/Stop-xDrip.html  
Then, you can use Dexcom app.
If you can show that Dexcom app works on the exact same phone with no intermittent issues and xDrip keeps disconnecting, we will have something to work on.  Without that, the only conclusion I can make is that this is expected behavior from Xiaomi.  

It makes no differences that it worked fine before.  It is not working now and the reason is your phone.

**Comment by kresimirjedud** [comment]
I will try to install the Dex app tomorrow, for the time beeing I managed
to get the readings by charging the battery and deleting the Dex sensor
from the phone BT settings and changing and returning the receiver id in
the xDrip. This implicates that it could be connected to the phone issue. I
suspect to some strange behavior caused by low battery (was only 29%) -
eventhough the battery optimisation is off but I guess there might be some
hidden settings that cannot be cleared. Maybe I'm wrong, I don't know. Will
follow the situation and report for the changes that might be usefull to
the community.
Thank you for your assist!


uto, 13. kol 2024. 23:08 Navid ***@***.***> je napisao:
